hf_xet/src/logging.rs

```diff
@@ -83,11 +85,41 @@ fn init_logging_to_file(path: &Path) -> Result<(), std::io::Error> {
     Ok(())
 }
 
-fn init_global_logging(py: Python) -> Option<TelemetryTaskInfo> {
+fn get_version_info_string(py: Python<'_>) -> String {
+    // populate remote telemetry calls with versions for python and hf_hub if possible
+    let mut version_info = String::new();
+
+    // Get Python version
+    if let Ok(sys) = py.import("sys") {
+        if let Ok(version) = sys.getattr("version").and_then(|v| v.extract::<String>()) {
+            if let Some(python_version_number) = version.split_whitespace().next() {
+                version_info.push_str(&format!("python/{python_version_number}; "));
+            }
+        }
+    }
+
+    // Get huggingface_hub+hf_xet versions
+    let package_names = ["huggingface_hub", "hfxet"];
+    if let Ok(importlib_metadata) = py.import("importlib.metadata") {
+        for package_name in package_names.iter() {
+            if let Ok(version) = importlib_metadata
+                .call_method1("version", (package_name,))
+                .and_then(|v| v.extract::<String>())
+            {
+                version_info.push_str(&format!("{package_name}/{version}; "));
+            }
+        }
+    }
+    version_info
+}
+
+fn init_global_logging(py: Python) {
+    let version_info = get_version_info_string(py);
```

Comment:
nit: can we push this down to only be done if we have telemetry enabled (i.e. move to line 140 inside the HF_HUB_ENABLE_TELEMETRY block)?

Reply:
Ah!  I realized in other places we never record this information, which would be useful for debugging, so I was going to add an info! statement to the other paths to log this.  Put those in.